iOS/UI/Manga/MangaViewController.swift

```diff
@@ -444,6 +444,18 @@ class MangaViewController: BaseTableViewController {
         present(UIHostingController(rootView: SwiftUINavigationView(rootView: AnyView(migrateView))), animated: true)
     }
 
+    func showNoWifiAlert() {
+        let alertController = UIAlertController(
+            title: NSLocalizedString("NO_WIFI_ALERT_TITLE", comment: ""),
+            message: NSLocalizedString("NO_WIFI_ALERT_MESSAGE", comment: ""),
+            preferredStyle: .alert)
+
+        let okAction = UIAlertAction(title: NSLocalizedString("OK", comment: ""), style: .default)
+        alertController.addAction(okAction)
+
+        present(alertController, animated: true, completion: nil)
```

Comment:
I don't think you need the completion parameter since it defaults to nil
```suggestion
        present(alertController, animated: true)
```